src/main/scala/no/ndla/listingapi/model/domain/search/SearchableCover.scala

```diff
@@ -32,5 +32,6 @@ case class SearchableCover(
   supportedLanguages: Seq[String],
   updatedBy: String,
   update: Date,
-  theme: String
+  theme: String,
+  oldNodeId: Long
```

Comment:
Den er med i søket for å at vi skal kunne lage oembedUrlen i søk responsen, ellers blir det en nullpointer her: https://github.com/NDLANO/listing-api/blob/master/src/main/scala/no/ndla/listingapi/service/search/SearchService.scala#L86

Mulig det er noe jeg har missforstått med hvordan ES fungerer her.